spras/mincostflow.py

```diff
@@ -24,6 +24,7 @@
 """
 class MinCostFlow (PRM):
     required_inputs = ['sources', 'targets', 'edges']
+    dois = ["10.1038/s41540-020-00167-1"]
```

Comment:
The mincostflow code wasn't created based on this paper. It gets it's inspiration from the ResponseNet paper. and uses Google's OR-Tools library: https://developers.google.com/optimization/flow/mincostflow.

Reply:
Hm - I could drop `dois` from this PR for now, since it seems that citing that library would require more rich information than a DOI. Or, I could use `10.4230/LIPIcs.CP.2023.3`